### PR TITLE
Change datepicker to have default time

### PR DESCRIPTION
**Pull request by aneshodza**
This PR allows date input to be typed and also sets the default datetime to now without populating the field
image

### CODE REVIEW

**Comment by rnestler** [comment]
Part of it is already done in #64

**Comment by aneshodza** [comment]
> Part of it is already done in https://github.com/renuo/redmine_tracky/pull/64

Removed that

File: app/views/shared/_time_entry_input.html.slim [hunk 1]
```diff
@@ -6,6 +6,8 @@
       flatpickr_minute_Increment: 1,
       flatpickr_enable_time: true,
       flatpickr_time_24hr: true,
+      flatpickr_default_hour: Time.now.hour,
+      flatpickr_default_minute: Time.now.min,
```
Comment (rnestler) [inline app/views/shared/_time_entry_input.html.slim:10]: Does this work even if the page has been running for a while? I'd guess this is just initialized on page load, right?

Comment (rnestler) [reply app/views/shared/_time_entry_input.html.slim:10]: https://github.com/renuo/redmine_tracky/pull/67 does it in JS

Comment (aneshodza) [reply app/views/shared/_time_entry_input.html.slim:10]: Yes its only on page load. Should I look for a way that updates it constantly?

**Pull request closed without merging**